conda/petsc/build.sh

```diff
@@ -1,119 +1,164 @@
 #!/bin/bash
-set -eu
-export PATH=/bin:$PATH
-
+#
```

Comment:
This is some awesome culling of unnecessary stuff! That being said, we should remove the comments.